dev/com.ibm.ws.jdbc_fat/fat/src/com/ibm/ws/jdbc/fat/tests/ConfigTest.java

```diff
@@ -372,6 +372,25 @@ public void testConfigChangeConnectionManager() throws Throwable {
             throw x;
         }
 
+        runTest(basicfat, "setServletInstanceStillActive");
+
+        // Increase maxPoolSize to 2
+        conMgr2.setMaxPoolSize("2");
+
+        try {
+            updateServerConfig(config, EMPTY_EXPR_LIST);
+            // Behavior should now reflect the new setting of 1 for maxPoolSize
```

Comment:
Change 1 -> 2 for maxPoolSize comment

Reply:
Thanks for catching that. I added a commit to fix the comment.